### ISSUE TITLE
After training for 20 epochs with default params, the validation results seem to be quite blurry.

### ISSUE THREAD

**Issue by Crestina2001** [jhhuang96/ConvLSTM-PyTorch#10]
![image](https://github.com/jhhuang96/ConvLSTM-PyTorch/assets/87475993/545cb796-3090-48af-b573-78d87b7a432e)
Could anyone offer a solution. Many thanks.

**Comment by Crestina2001** [comment]
The config is like:

convlstm: False
convgru: False
batch_size: 4
lr: 0.0001
frames_input: 10
frames_output: 10
epochs: 20


When convlstm and convgru are both False, it actually uses convgru.

**Comment by Crestina2001** [comment]
Here is the result using convlstm(epochs: 20):
![image](https://github.com/jhhuang96/ConvLSTM-PyTorch/assets/87475993/c2bee38c-608e-4c9c-afea-b0ff0cb598d6)

**Comment by Crestina2001** [comment]
After training for 150 epochs, the result seem to improve, but not by much. I get the best val loss when epoch = 144. Here is the visualization:
![validation_sample_3](https://github.com/jhhuang96/ConvLSTM-PyTorch/assets/87475993/3eddb936-e5de-4b64-9e70-0438dab5c5f4)
It is still far from satisfaction.

**Comment by contain-mm** [comment]
guru,hello!Could you share your code with me.Great thanks!
